integration/kubernetes/confidential/agent_image.bats

```diff
@@ -164,7 +164,10 @@ setup() {
 
 	assert_pod_fail "${pod_config}"
 
-	assert_logs_contain "containerd" 'failed to resolve reference \\"quay.io/kata-containers/confidential-containers-auth:test\\": pulling from host quay.io failed with status code \[manifests test\]: 401 UNAUTHORIZED'
+	# The error will be different depending on the containerd version being used.
+	# The former assert is the one coming from containerd 1.6.x, while the latter is the cone coming from 1.7.x
```

Comment:
nit: "one"

Reply:
As you marked it as a nit, I will save some trees and avoid re-running the CIs as they take a at least an hour, thanks for the review though